### ISSUE TITLE
what is the meaning of 'RootTransformation' in smpl_params.txt?

### ISSUE THREAD

**Issue by xjturobocon** [ZhengZerong/DeepHuman#1]
Hi, 
Great job! I use the pose and shape params(72+10) of SMPL model in the smpl_params.txt to generate the SMPL mesh, but I find it is different with the one provided by you. I try to use the 'RootTransformation' params to rotate and translate the generated SMPL mesh, after the transformation, It seems that it is same with the provided one in rotation, but still exists translation difference.  Can you kindly tell me how you use the pose and shape params to generate the SMPL mesh? and how you use the 'RootTransformation' params?
Thanks.

**Comment by ZhengZerong** [comment]
Hi, thanks for pointing out. The rotation in 'RootTransformation' params means the rotation of the model around the **original point** of the coordinate system. This is different from the SMPL official code sample, in which the model is rotated around the **root** joint. I guess this is the reason why you get different results. I have provided a piece of code in dataset readme to show how to use the parameters we provided. Thanks.
